### ISSUE TITLE
(OUI Next Theme) Observability Logs

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/dashboards-observability#766]
_This issue to be transferred to corresponding repository_

I am working on launching new light and dark mode themes provided by [OUI component library](https://oui.opensearch.org/1.2/#/) for a target launch within 2.10. These changes support the vision expressed in [Future Vision for Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4298.)

I have identified the following front end related issues that prevent the theme from appearing complete and potential solutions within this feature:

- [ ] globalQueryBar - PPL link should utilize `$ouiColorPrimary` in favor of hard coded values
- [ ] globalQueryBar - there is custom coloring applied to the "dropdown" sections in the query builder. Consider replacing with Sass variables (they adjust appropriately across theme & modes)  or migrating to `EuiSuggest`

**Comment by pjfitzgibbons** [comment]
@KrooshalUX for " "dropdown" sections" ...
Do you mean this? :
![image](https://github.com/opensearch-project/dashboards-observability/assets/11318/b4852682-0cc8-4935-8886-d1e75fcd2a96)

**Comment by KrooshalUX** [comment]
Yes. Those values appear custom / hard coded and not coming from OUI, which means they do not translate within the Next theme dark mode. 
<img width="532" alt="Screenshot 2023-08-31 at 5 37 32 PM" src="https://github.com/opensearch-project/dashboards-observability/assets/105819041/b8b4fb7f-d2cc-4be0-8673-f2129d87ebad">

**Comment by joshuarrrr** [comment]
If you need help from the dashboards team to resolve, feel free to sign up for OpenSearch Dashboards developer office hours: https://opensearch.org/events/2023-0921-dev-officehours-dashboards/
